proto/api/users.proto

```diff
@@ -1,10 +1,13 @@
 syntax = "proto3";
 
 package api;
+
+option go_package = "gohire/proto/gen/api";
```

Comment:
cool, I think this fixed a bug on why buf generate was not updated the generated files?

Reply:
I actually had a lot of trouble getting buf to generate/compile on my machine and ultimately went with protoc commands utilizing two different plugins